src/toil_scripts/lib/jobs.py

```diff
@@ -0,0 +1,20 @@
+from toil_scripts.lib import partitions
+
+
+def sample_batcher_job(job, func, samples, *args):
+    """
+    Spawns a tree of jobs to avoid overloading the number of jobs spawned by a single parent.
+
+    :param function func: Function to spawn dynamically, passes one sample as first argument
+    :param list samples: Array of samples to be batched
+    :param list args: any arguments to be passed to the function
+    :return:
+    """
+    num_partitions = 100
```

Comment:
Unless this number is a fixed constraint in toil, I would expose this as a parameter. At the very least, I'd add a comment to explain why 100 was chosen.